source/uwp/SharedVisualizer/ViewModel/HostConfigEditorViewModel.cs

```diff
@@ -58,9 +59,10 @@ protected override void LoadPayload(string payload)
 
         public static async Task<HostConfigEditorViewModel> LoadAsync(MainPageViewModel mainPageViewModel)
         {
+            string fileName = PickHostConfigFile();
             try
             {
-                StorageFile file = await StorageFile.GetFileFromApplicationUriAsync(new Uri("ms-appx:///HostConfigs/DefaultHostConfig.json"));
+                StorageFile file = await StorageFile.GetFileFromApplicationUriAsync(new Uri("ms-appx:///" + fileName));
```

Comment:
super nitpicky, but this might be a little cleaner if you left in `HostConfigs/` and only appended the filename... feel free to not take this suggestion. :)